fiat-google-groups/src/main/java/com/netflix/spinnaker/fiat/roles/google/GoogleDirectoryUserRolesProvider.java

```diff
@@ -174,8 +174,23 @@ public void onSuccess(Groups groups, HttpHeaders responseHeaders) throws IOExcep
   }
 
   protected Groups getGroupsFromEmail(String email) throws IOException {
-    Directory service = getDirectoryService();
-    return service.groups().list().setDomain(config.getDomain()).setUserKey(email).execute();
+    final Directory service = getDirectoryService();
+    final Groups groups =
+        service.groups().list().setDomain(config.getDomain()).setUserKey(email).execute();
+    String nextPageToken = groups.getNextPageToken();
+    while (nextPageToken != null) {
+      final Groups nextPage =
+          service
+              .groups()
+              .list()
+              .setDomain(config.getDomain())
+              .setUserKey(email)
+              .setPageToken(nextPageToken)
+              .execute();
+      groups.getGroups().addAll(nextPage.getGroups());
```

Comment:
I don't know that you could rely on the List<Group> from getGroups being mutable. Might be worth keeping a separate List<Group> that you accumulate into and then call groups.setGroups with that at the end just in case

Reply:
Good point.
I'm going to work on this and update the PR if needed.

Thanks for the review btw.

Reply:
I tested and it works fine, so the list from getGroups is mutable.

Reply:
@cfieber Are you ok with the current code? Or do you want to change it?